### DISCUSSION TITLE
Is there any difference if i download precompiled wine and compress it

### DISCUSSION THREAD

**Discussion by xDoge26** [Kron4ek/Wine-Builds#80] [General]
I can't compile wine with those scripts

**Comment by Kron4ek** [comment]
You mean the difference between precompiled builds and manually compiling from source code?

**Reply by xDoge26** [reply]
> You mean the difference between precompiled builds and manually compiling from source code?

yep

**Reply by Kron4ek** [reply]
If the source code, the build environment and CFLAGS are the same as those that were used for the precompiled builds, there is no difference. And in your case there is no difference, considering that you tried to compile in the same chroots that i use and didn't change CFLAGS.
